Misc/verify-bundled-wheels.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
```

Comment:
Is there a reason this is written as a shell script? If (as @hugovk mentioned) it would be useful to run locally, rewriting in Python would make this more portable cross-platform (e.g. Windows).

Happy to help rewriting, just don't want to do the work if there's a requirement to have it be a shell script I'm missing!

A

Reply:
> Is there a reason this is written as a shell script?

[Initially](https://github.com/python/cpython/pull/31885/commits/e4a8faf7b4e4aeb553e246d7e58d887a018dc341#diff-d7623692dd445a8dd8e63aaa1a557c50c8cd4e530aee2634865caa0b402a69a8) this script was embedded in the workflow file, making it a shell script seemed to be cleaner at the time.

Reply:
I think this can be a Python script, but changing over to that should not be a blocking concern for the PR. :)